pom.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>2.1.0</version>
+  <version>2.1.0-SNAPSHOT</version>
```

Comment:
It should be 2.1.1-SNAPSHOT, and please update the pom file in e2e test folder as well

Reply:
@sfc-gh-tzhang addressed PR comments.